### PR TITLE
Register parameters of SVM

### PR DESCRIPTION
**Pull request by vinx13**

### CODE REVIEW

File: shogun/classifier/svm/SVMLin.cpp [hunk 1]
```diff
@@ -30,13 +31,26 @@ CSVMLin::CSVMLin(
...
 
 
 CSVMLin::~CSVMLin()
 {
 }
 
+void CSVMLin::init()
+{
+	SG_ADD(
+	    &use_bias, "use_bias", "Indicates if bias is used.", MS_NOT_AVAILABLE);
```
Comment (karlnapf) [inline shogun/classifier/svm/SVMLin.cpp:46]: I wonder: are those things not registered in the base class?
If not why not?

File: shogun/classifier/svm/SVMSGD.cpp [hunk 2]
```diff
@@ -214,13 +213,16 @@ void CSVMSGD::init()
...
-    m_parameters->add(&C1, "C1",  "Cost constant 1.");
-    m_parameters->add(&C2, "C2",  "Cost constant 2.");
-    m_parameters->add(&wscale, "wscale",  "W scale");
-    m_parameters->add(&bscale, "bscale",  "b scale");
-    m_parameters->add(&epochs, "epochs",  "epochs");
-    m_parameters->add(&skip, "skip",  "skip");
-    m_parameters->add(&count, "count",  "count");
-    m_parameters->add(&use_bias, "use_bias",  "Indicates if bias is used.");
-    m_parameters->add(&use_regularized_bias, "use_regularized_bias",  "Indicates if bias is regularized.");
+	SG_ADD(&C1, "C1", "Cost constant 1.", MS_AVAILABLE);
```
Comment (karlnapf) [inline shogun/classifier/svm/SVMSGD.cpp:216]: because then we need to register them everytime ...

Comment (vinx13) [reply shogun/classifier/svm/SVMSGD.cpp:216]: actually they don't inherit from SVM base class

Comment (karlnapf) [reply shogun/classifier/svm/SVMSGD.cpp:216]: seems quite mad....
can you see any idea why that is?
On the other hand, you are just porting examples, so no need to refactor things ....

File: shogun/classifier/svm/SVMSGD.cpp [hunk 3]
```diff
@@ -214,13 +213,16 @@ void CSVMSGD::init()
...
+	SG_ADD(&C2, "C2", "Cost constant 2.", MS_AVAILABLE);
+	SG_ADD(&wscale, "wscale", "W scale", MS_NOT_AVAILABLE);
```
Comment (karlnapf) [inline shogun/classifier/svm/SVMSGD.cpp:218]: are these parameters necessary to be accessible from the outside/equals/serilazation, or are they only to store the intermediate state of training?

**Comment by karlnapf** [comment]
anyways, will merge for now and we can come back to it

**Pull request merged**